template/src/main/scala/ch.epfl.scala.index.views/package.scala

```diff
@@ -108,7 +108,8 @@ package object html {
     val prefix = "scala_"
     project.targets
       .collect { case t if t.startsWith(prefix) => t.drop(prefix.length) }
-      .to[List].sorted(Ordering.String.reverse)
+      .map(SemanticVersion.apply).flatten
```

Comment:
Alright, you should actually write it a little bit differently in order to get the implicit conversion triggered, but that doesn’t really matter, let’s merge it as it is now!